mobile/examples/whisper/azure/android/app/src/main/java/ai/onnxruntime/example/whisperAzure/AudioTensorSource.kt

```diff
@@ -0,0 +1,202 @@
+package ai.onnxruntime.example.whisperAzure
+
+import ai.onnxruntime.OnnxJavaType
+import ai.onnxruntime.OnnxTensor
+import ai.onnxruntime.OrtEnvironment
+import android.annotation.SuppressLint
+import android.content.Context
+import android.media.AudioFormat
+import android.media.AudioRecord
+import android.media.MediaRecorder
+import android.util.Log
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+import java.util.concurrent.atomic.AtomicBoolean
+import java.io.DataOutputStream
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.IOException
+import java.io.FileOutputStream
+
+class AudioTensorSource {
+    companion object {
+        private const val sampleRate = 16000
+        private const val maxAudioLengthInSeconds = 30
+
+        private fun readWavFile(filePath: String): ByteArray {
+            val file = File(filePath)
+            if (!file.exists()) {
+                throw FileNotFoundException("Unable to fild output wav audio file with the specified path.")
+            }
+
+            var inputStream: FileInputStream? = null
+            try {
+                inputStream = FileInputStream(file)
+                val fileLength = file.length().toInt()
+                val fileData = ByteArray(fileLength)
+
+                inputStream.read(fileData)
+                return fileData
+            } finally {
+                inputStream?.close()
+            }
```

Comment:
consider [`use`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html) instead of try/finally
https://kotlinlang.org/docs/idioms.html#java-7-s-try-with-resources